tgui/packages/tgui/interfaces/Soulcatcher.js

```diff
@@ -0,0 +1,271 @@
+                            </Button>
+                          </LabeledList.Item>
+                          <LabeledList.Item label="Speech">
+                            <Button
+                              color={soul.able_to_speak ? 'green' : 'red'}
+                              fluid
+                              tooltip="Is the soul able to speak?"
+                              onClick={() =>
+                                act('toggle_soul_communication', {
+                                  target_soul: soul.reference,
+                                  communication_type: 'speech',
+                                  room_ref: room.reference,
+                                })
+                              }>
+                              {soul.able_to_speak ? 'Enabled' : 'Disabled'}
+                            </Button>
+                          </LabeledList.Item>
+                          <LabeledList.Item label="Emote">
+                            <Button
+                              color={soul.able_to_emote ? 'green' : 'red'}
+                              fluid
+                              tooltip="Is the soul able to emote?"
+                              onClick={() =>
+                                act('toggle_soul_communication', {
+                                  target_soul: soul.reference,
+                                  communication_type: 'emote',
+                                  room_ref: room.reference,
+                                })
+                              }>
+                              {soul.able_to_emote ? 'Enabled' : 'Disabled'}
+                            </Button>
+                          </LabeledList.Item>
+                        </LabeledList>
+                        <br />
+                        <Button
+                          fluid
+                          icon="eject"
+                          color="red"
+                          onClick={() =>
+                            act('remove_soul', {
+                              target_soul: soul.reference,
+                              room_ref: room.reference,
+                            })
+                          }>
+                          Remove Soul
+                        </Button>
+                      </Collapsible>
+                    </Flex.Item>
+                  ))}
+                </Flex>
+              </>
+            ) : (
+              <> </>
+            )}
+          </Section>
+        ))}
+        <Button
+          fluid
+          color="green"
+          icon="plus"
+          onClick={() => act('create_room', {})}>
+          Create new room
+        </Button>
+        <Button
+          fluid
+          color={ghost_joinable ? 'green' : 'red'}
+          icon={ghost_joinable ? 'door-open' : 'door-closed'}
+          onClick={() => act('toggle_joinable', {})}>
+          {ghost_joinable ? 'Opened' : 'Closed'} to ghosts
```

Comment:
i think golden made a misatke with their suggestion(?)

the code escaped the bracket

```suggestion
          {ghost_joinable ? 'Opened' : 'Closed' to ghosts}
```

Reply:
also it looks like the "to ghosts" escaped the quote as well